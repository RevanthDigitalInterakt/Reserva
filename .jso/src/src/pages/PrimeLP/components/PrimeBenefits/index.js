  Object.defineProperty(exports, "__esModule", {
    value: true
  });
  Object.defineProperty(exports, "default", {
    enumerable: true,
    get: function get() {
      return _PrimeBenefits.default;
    }
  });
  var _PrimeBenefits = _$$_REQUIRE(_dependencyMap[0])(_$$_REQUIRE(_dependencyMap[1]));
