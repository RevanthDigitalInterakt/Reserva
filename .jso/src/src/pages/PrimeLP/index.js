  Object.defineProperty(exports, "__esModule", {
    value: true
  });
  Object.defineProperty(exports, "default", {
    enumerable: true,
    get: function get() {
      return _PrimeLP.default;
    }
  });
  var _PrimeLP = _$$_REQUIRE(_dependencyMap[0])(_$$_REQUIRE(_dependencyMap[1]));
