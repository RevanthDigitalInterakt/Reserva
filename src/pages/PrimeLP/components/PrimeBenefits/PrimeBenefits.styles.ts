import { StyleSheet } from 'react-native';
import { COLORS } from '../../../../base/styles/colors';

export const styles = StyleSheet.create({
  wrapper: {
    backgroundColor: '#F7F7F7',
    alignItems: 'center',
    justifyContent: 'center',
    paddingHorizontal: 32,
    paddingBottom: 8,
  },

  title: {
    fontSize: 20,
    lineHeight: 26,
    marginBottom: 24,
  },

  titleBold: {
    fontSize: 20,
  },

  subtitle: {
    fontSize: 16,
    lineHeight: 20,
    textAlign: 'center',
    color: COLORS.GRAY,
    marginBottom: 32,
  },

  listWrapper: {
    paddingHorizontal: 20,
  },

  itemContainer: {
    alignItems: 'center',
    justifyContent: 'center',
    marginBottom: 32,
  },

  icon: {
    marginBottom: 32,
  },

  itemTitle: {
    marginBottom: 16,
    fontSize: 20,
    lineHeight: 20,
    textAlign: 'center',
  },

  itemDescription: {
    color: COLORS.GRAY,
    fontSize: 16,
    lineHeight: 20,
    textAlign: 'center',
  },

  itemDescriptionBold: {
    color: '#000000',
    fontSize: 16,
    lineHeight: 20,
  },
});
