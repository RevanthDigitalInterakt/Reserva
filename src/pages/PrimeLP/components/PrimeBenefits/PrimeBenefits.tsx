import React, { useMemo } from 'react';
import { View } from 'react-native';
import IconPrimeDiscount from '../Icons/IconPrimeDiscount';
import IconPrimeFreeShipping from '../Icons/IconPrimeFreeShipping';
import IconPrimeCashback from '../Icons/IconPrimeCashback';
import { styles } from './PrimeBenefits.styles';
import type { PrimeDetailOutput } from '../../../../base/graphql/generated';
import IconPrimePartners from '../Icons/IconPrimePartners';
import { Typography } from '../../../../components/Typography/Typography';

interface IPrimeBenefits {
  data: PrimeDetailOutput;
}

function PrimeBenefits({ data }: IPrimeBenefits) {
  const benefits = useMemo(() => ([
    {
      Icon: <IconPrimeDiscount style={styles.icon} />,
      title: 'Descontos Especiais',
      text: (
        <Typography variant="tituloSessao" style={styles.itemDescription}>
          <Typography variant="precoTotal" style={styles.itemDescriptionBold}>15% OFF* </Typography>
          em compras a partir de
          <Typography variant="precoTotal" style={styles.itemDescriptionBold}> R$ 499 </Typography>
          em todas as peças das marcas Reserva.
        </Typography>
      ),
    },
    {
      Icon: <IconPrimeFreeShipping style={styles.icon} />,
      title: 'Frete Grátis',
      text: (
        <Typography variant="tituloSessao" style={styles.itemDescription}>
          Em toda compra*, sem valor mínimo e
          válido em todo o Brasil.
        </Typography>
      ),
    },
    {
      Icon: <IconPrimeCashback style={styles.icon} />,
      title: `R$ ${data.monthlyCashback} de Cashback Mensal`,
      text: (
        <Typography variant="tituloSessao" style={styles.itemDescription}>
          <Typography variant="precoTotal" style={styles.itemDescriptionBold}>
            Acumule mais*
          </Typography>
          pra comprar o que você quiser, quando preferir.
        </Typography>
      ),
    },
    {
      Icon: <IconPrimePartners style={styles.icon} />,
      title: 'Programa de Parcerias',
      text: (
        <Typography variant="tituloSessao" style={styles.itemDescription}>
          <Typography variant="precoTotal" style={styles.itemDescriptionBold}>
            Acesso exclusivo
          </Typography>
          {' '}
          a benefícios com marcas parceiras de variados segmentos.
        </Typography>
      ),
    },
  ]), [data]);

  return (
    <View style={styles.wrapper} testID="com.usereserva:id/PrimeBenefits_wrapper">
      <Typography variant="tituloSessoes" style={styles.title}>
        Vantagens em ser
        <Typography variant="descontoTag2" style={styles.titleBold}>{' Prime'}</Typography>
      </Typography>

      <Typography variant="tituloSessao" style={styles.subtitle}>
        Ganhe um infinito de comodidades pra aproveitar no site, App,
        lojas físicas e até fora da Reserva. Além de acesso a
        pré-lançamentos e produtos exclusivos, você também garante:
      </Typography>

      <View style={styles.listWrapper}>
        {benefits.map((item) => (
          <View key={`prime-benefit-${item.title}`} style={styles.itemContainer}>
            {item.Icon}

            <Typography variant="descontoTag2" style={styles.itemTitle}>
              {item.title}
            </Typography>

            {item.text}
          </View>
        ))}
      </View>
    </View>
  );
}

export default PrimeBenefits;
