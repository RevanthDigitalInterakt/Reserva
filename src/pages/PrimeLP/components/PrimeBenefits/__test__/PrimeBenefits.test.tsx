import { render, screen } from '@testing-library/react-native';
import React from 'react';
import { ThemeProvider } from 'styled-components/native';

import { mockPrimeData } from '../../../../../../__mocks__/PrimeLP.mock';
import { theme } from '../../../../../base/usereservappLegacy/theme';
import PrimeBenefits from '../PrimeBenefits';

const TestingComponent = (
  <ThemeProvider theme={theme}>
    <PrimeBenefits data={mockPrimeData} />
  </ThemeProvider>
);

describe('PrimeSubscribe', () => {
  beforeEach(() => {
    render(TestingComponent);
  });
  it('should render properly and match with snapshot', () => {
    const primeSubscribe = screen.getByTestId(
      'com.usereserva:id/PrimeBenefits_wrapper',
    );

    expect(primeSubscribe).toBeOnTheScreen();
    expect(screen.toJSON()).toMatchSnapshot();
  });
});
