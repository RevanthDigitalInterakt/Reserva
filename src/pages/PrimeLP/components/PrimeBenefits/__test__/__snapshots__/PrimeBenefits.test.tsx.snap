// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrimeSubscribe should render properly and match with snapshot 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#F7F7F7",
      "justifyContent": "center",
      "paddingBottom": 8,
      "paddingHorizontal": 32,
    }
  }
  testID="com.usereserva:id/PrimeBenefits_wrapper"
>
  <Text
    style={
      [
        {
          "fontFamily": "ReservaSerif-Regular",
          "fontSize": 28,
        },
        {
          "fontSize": 20,
          "lineHeight": 26,
          "marginBottom": 24,
        },
      ]
    }
    variant="tituloSessoes"
  >
    Vantagens em ser
    <Text
      style={
        [
          {
            "fontFamily": "ReservaSerif-Bold",
            "fontSize": 15,
          },
          {
            "fontSize": 20,
          },
        ]
      }
      variant="descontoTag2"
    >
       Prime
    </Text>
  </Text>
  <Text
    style={
      [
        {
          "fontFamily": "Nunito-Regular",
          "fontSize": 15,
        },
        {
          "color": "#848484",
          "fontSize": 16,
          "lineHeight": 20,
          "marginBottom": 32,
          "textAlign": "center",
        },
      ]
    }
    variant="tituloSessao"
  >
    Ganhe um infinito de comodidades pra aproveitar no site, App, lojas físicas e até fora da Reserva. Além de acesso a pré-lançamentos e produtos exclusivos, você também garante:
  </Text>
  <View
    style={
      {
        "paddingHorizontal": 20,
      }
    }
  >
    <View
      style={
        {
          "alignItems": "center",
          "justifyContent": "center",
          "marginBottom": 32,
        }
      }
    >
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight="41"
        bbWidth="54"
        fill="none"
        focusable={false}
        height={41}
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          [
            {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            {
              "marginBottom": 32,
            },
            {
              "flex": 0,
              "height": 41,
              "width": 54,
            },
          ]
        }
        vbHeight={41}
        vbWidth={54}
        width={54}
      >
        <RNSVGGroup
          fill={null}
          propList={
            [
              "fill",
            ]
          }
        >
          <RNSVGPath
            d="M43.207 1.008s9.62-.51 9.74 6.498c.12 7.008 0 25.991 0 25.991s.175 6.498-6.494 6.498H1V27s6.494-.165 6.494-6.498S1 14.004 1 14.004V1.008h42.207Z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4282006074,
                "type": 0,
              }
            }
            strokeWidth="2"
          />
          <RNSVGPath
            d="M17.234 7.506v7.38M17.234 16.811v7.38M17.234 26.117v7.38"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4290513165,
                "type": 0,
              }
            }
            strokeWidth="2"
          />
          <RNSVGPath
            d="m26.974 30.248 16.233-19.493M28.658 19.308a3.716 3.716 0 0 0 3.715-3.717 3.716 3.716 0 0 0-3.715-3.718 3.716 3.716 0 0 0-3.715 3.718 3.716 3.716 0 0 0 3.715 3.717ZM41.24 28.901a3.716 3.716 0 0 0 3.715-3.717 3.716 3.716 0 1 0-7.43 0 3.716 3.716 0 0 0 3.715 3.717Z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4282006074,
                "type": 0,
              }
            }
            strokeWidth="2"
          />
        </RNSVGGroup>
      </RNSVGSvgView>
      <Text
        style={
          [
            {
              "fontFamily": "ReservaSerif-Bold",
              "fontSize": 15,
            },
            {
              "fontSize": 20,
              "lineHeight": 20,
              "marginBottom": 16,
              "textAlign": "center",
            },
          ]
        }
        variant="descontoTag2"
      >
        Descontos Especiais
      </Text>
      <Text
        style={
          [
            {
              "fontFamily": "Nunito-Regular",
              "fontSize": 15,
            },
            {
              "color": "#848484",
              "fontSize": 16,
              "lineHeight": 20,
              "textAlign": "center",
            },
          ]
        }
        variant="tituloSessao"
      >
        <Text
          style={
            [
              {
                "fontFamily": "Nunito-Bold",
                "fontSize": 20,
              },
              {
                "color": "#000000",
                "fontSize": 16,
                "lineHeight": 20,
              },
            ]
          }
          variant="precoTotal"
        >
          15% OFF* 
        </Text>
        em todas as peças das marcas Reserva.
      </Text>
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "justifyContent": "center",
          "marginBottom": 32,
        }
      }
    >
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight="58"
        bbWidth="50"
        fill="none"
        focusable={false}
        height={58}
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          [
            {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            {
              "marginBottom": 32,
            },
            {
              "flex": 0,
              "height": 58,
              "width": 50,
            },
          ]
        }
        vbHeight={58}
        vbWidth={50}
        width={50}
      >
        <RNSVGGroup
          fill={null}
          propList={
            [
              "fill",
            ]
          }
        >
          <RNSVGPath
            d="M1.312 12.347v27.146l23.453 10.93L48.3 39.493V12.347L24.765 2 1.312 12.347Z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4282006074,
                "type": 0,
              }
            }
            strokeWidth="2"
          />
          <RNSVGPath
            d="M1.949 12.681 24.764 24.92 48.693 12.68M24.764 50.423V24.92"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4282006074,
                "type": 0,
              }
            }
            strokeWidth="2"
          />
          <RNSVGPath
            d="m1 18.288 23.765 12.48L49 18.289"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4282006074,
                "type": 0,
              }
            }
            strokeWidth="2"
          />
          <RNSVGPath
            d="M12.703 9.22 33.61 19.775v12.433l4.116-4.758 3.848 2.199V15.97L20.7 5.706 12.703 9.22Z"
            fill={
              {
                "payload": 4290513165,
                "type": 0,
              }
            }
            propList={
              [
                "fill",
                "stroke",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
          />
          <RNSVGPath
            d="M24.843 57c3.602 0 6.522-3.093 6.522-6.907 0-3.815-2.92-6.908-6.522-6.908-3.601 0-6.521 3.093-6.521 6.907 0 3.815 2.92 6.908 6.521 6.908Z"
            fill={
              {
                "payload": 4290513165,
                "type": 0,
              }
            }
            propList={
              [
                "fill",
              ]
            }
          />
          <RNSVGPath
            d="M24.843 56.232c3.178 0 5.755-2.749 5.755-6.14 0-3.39-2.577-6.14-5.755-6.14s-5.754 2.75-5.754 6.14c0 3.391 2.576 6.14 5.754 6.14Z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth="2"
          />
          <RNSVGPath
            d="m22.474 49.773 1.784 1.95 2.88-3.736"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
          />
        </RNSVGGroup>
      </RNSVGSvgView>
      <Text
        style={
          [
            {
              "fontFamily": "ReservaSerif-Bold",
              "fontSize": 15,
            },
            {
              "fontSize": 20,
              "lineHeight": 20,
              "marginBottom": 16,
              "textAlign": "center",
            },
          ]
        }
        variant="descontoTag2"
      >
        Frete Grátis
      </Text>
      <Text
        style={
          [
            {
              "fontFamily": "Nunito-Regular",
              "fontSize": 15,
            },
            {
              "color": "#848484",
              "fontSize": 16,
              "lineHeight": 20,
              "textAlign": "center",
            },
          ]
        }
        variant="tituloSessao"
      >
        Em toda compra*, sem valor mínimo e válido em todo o Brasil.
      </Text>
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "justifyContent": "center",
          "marginBottom": 32,
        }
      }
    >
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight="47"
        bbWidth="54"
        fill="none"
        focusable={false}
        height={47}
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          [
            {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            {
              "marginBottom": 32,
            },
            {
              "flex": 0,
              "height": 47,
              "width": 54,
            },
          ]
        }
        vbHeight={47}
        vbWidth={54}
        width={54}
      >
        <RNSVGGroup
          fill={null}
          propList={
            [
              "fill",
            ]
          }
        >
          <RNSVGPath
            d="M40 6.325H14L1 19.549l26 26.449 26-26.449L40 6.325Z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
                "strokeLinecap",
                "strokeLinejoin",
              ]
            }
            stroke={
              {
                "payload": 4282006074,
                "type": 0,
              }
            }
            strokeLinecap={1}
            strokeLinejoin={1}
            strokeWidth="2"
          />
          <RNSVGPath
            d="M1.392 20.08h50.689"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4282006074,
                "type": 0,
              }
            }
            strokeWidth="2"
          />
          <RNSVGPath
            d="M42.033 8.822 27 46 16.6 20.746 11.537 9.087"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
                "strokeLinecap",
                "strokeLinejoin",
              ]
            }
            stroke={
              {
                "payload": 4282006074,
                "type": 0,
              }
            }
            strokeLinecap={1}
            strokeLinejoin={1}
            strokeWidth="2"
          />
          <RNSVGPath
            d="M16.665 20.08 27 6.748 37.335 20.08"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
                "strokeLinecap",
                "strokeLinejoin",
              ]
            }
            stroke={
              {
                "payload": 4282006074,
                "type": 0,
              }
            }
            strokeLinecap={1}
            strokeLinejoin={1}
            strokeWidth="2"
          />
          <RNSVGPath
            d="M1 33.818s7.784 3.828 7.784 7.926c0-4.09 7.88-7.926 7.88-7.926s-7.88-3.746-7.88-8.469C8.784 30.171 1 33.82 1 33.82ZM38.306 6.939s5.458 2.684 5.458 5.556c0-2.867 5.526-5.556 5.526-5.556S43.764 4.315 43.764 1c0 3.38-5.458 5.939-5.458 5.939ZM38.306 29.947s4.226 2.078 4.226 4.303c0-2.22 4.278-4.303 4.278-4.303s-4.279-2.033-4.279-4.598c0 2.618-4.225 4.598-4.225 4.598Z"
            fill={
              {
                "payload": 4290513165,
                "type": 0,
              }
            }
            propList={
              [
                "fill",
                "stroke",
                "strokeWidth",
                "strokeLinecap",
                "strokeLinejoin",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeLinecap={1}
            strokeLinejoin={1}
            strokeWidth="2"
          />
        </RNSVGGroup>
      </RNSVGSvgView>
      <Text
        style={
          [
            {
              "fontFamily": "ReservaSerif-Bold",
              "fontSize": 15,
            },
            {
              "fontSize": 20,
              "lineHeight": 20,
              "marginBottom": 16,
              "textAlign": "center",
            },
          ]
        }
        variant="descontoTag2"
      >
        R$ 25 de Cashback Mensal
      </Text>
      <Text
        style={
          [
            {
              "fontFamily": "Nunito-Regular",
              "fontSize": 15,
            },
            {
              "color": "#848484",
              "fontSize": 16,
              "lineHeight": 20,
              "textAlign": "center",
            },
          ]
        }
        variant="tituloSessao"
      >
        <Text
          style={
            [
              {
                "fontFamily": "Nunito-Bold",
                "fontSize": 20,
              },
              {
                "color": "#000000",
                "fontSize": 16,
                "lineHeight": 20,
              },
            ]
          }
          variant="precoTotal"
        >
          Acumule mais*
        </Text>
        pra comprar o que você quiser, quando preferir.
      </Text>
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "justifyContent": "center",
          "marginBottom": 32,
        }
      }
    >
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight="43"
        bbWidth="68"
        fill="none"
        focusable={false}
        height={43}
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          [
            {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            {
              "marginBottom": 32,
            },
            {
              "flex": 0,
              "height": 43,
              "width": 68,
            },
          ]
        }
        vbHeight={43}
        vbWidth={68}
        width={68}
      >
        <RNSVGGroup
          fill={null}
          propList={
            [
              "fill",
            ]
          }
        >
          <RNSVGPath
            d="M4 5.001v-1 1Zm60 0v-1 1Zm-6 0v1-1Zm-39.261 6-.598-.801.598.802Zm0 5-.401.916.401-.916Zm9.261-1 .517.856.01-.006.011-.007-.538-.842Zm11 3-.788.616.788-.615Zm11 12 .642-.766-.008-.006-.634.773Zm-12.5-27.5-.275.96a1 1 0 0 0 .646-1.89l-.371.93ZM30 2l-.448-.893.448.894Zm-13 7v1-1Zm13 32-.142.99.142-.99Zm10-3 .858.514-.858-.513Zm5-2 .929.37-.929-.37Zm3.739-27-.633.775.633-.774Zm6.261 1h1v-1h-1v1Zm-47-5v1-1Zm27.52-3.066-.274.961.275-.961Zm11.054 33.047.031-1-.03 1Zm.847-.048-.176-.984.176.984Zm-33.59-4.932v1-1ZM12 9.002v21h2v-21h-2Zm-2 23H5v2h5v-2Zm-8-1v-23H0v23h2Zm3 1c-1.114 0-1.915.006-2.467-.173-.24-.078-.344-.167-.399-.239-.052-.07-.134-.226-.134-.589H0c0 .676.16 1.288.534 1.788.372.498.88.78 1.381.942.93.302 2.13.27 3.085.27v-2Zm7-2c0 .91-.207 1.347-.43 1.57-.225.224-.661.43-1.57.43v2c1.16 0 2.224-.257 2.982-1.014.76-.758 1.018-1.822 1.018-2.986h-2Zm-2-24c.67 0 1.12.287 1.443.774.35.528.557 1.317.557 2.226h2c0-1.168-.259-2.38-.89-3.332-.66-.994-1.71-1.668-3.11-1.668v2Zm-6-2c-1.16 0-2.224.256-2.982 1.013C.258 5.773 0 6.837 0 8.001h2c0-.91.207-1.347.43-1.57C2.656 6.207 3.092 6 4 6v-2Zm54 2h6v-2h-6v2Zm8 2v22h2v-22h-2Zm-3 24h-5v2h5v-2Zm-7-2v-22h-2v22h2Zm2 2c-.909 0-1.345-.206-1.57-.43-.223-.223-.43-.66-.43-1.57h-2c0 1.164.259 2.228 1.018 2.986.758.757 1.822 1.014 2.982 1.014v-2Zm8-2c0 .672-.289 1.122-.776 1.444-.53.35-1.318.556-2.224.556v2c1.163 0 2.374-.257 3.328-.888C67.323 32.455 68 31.404 68 30h-2Zm-2-24c.909 0 1.345.205 1.57.429.223.223.43.66.43 1.57h2c0-1.164-.259-2.228-1.018-2.986-.758-.757-1.822-1.014-2.982-1.014v2Zm-6-2c-1.16 0-2.224.256-2.982 1.013C54.258 5.773 54 6.837 54 8.002h2c0-.912.207-1.348.43-1.571.225-.224.661-.43 1.57-.43v-2ZM32.283 1.025c-.845.188-1.896.699-2.995 1.33-1.123.647-2.388 1.476-3.684 2.373-2.59 1.794-5.367 3.908-7.463 5.472l1.196 1.603c2.12-1.582 4.854-3.663 7.406-5.43 1.275-.884 2.488-1.677 3.543-2.284 1.08-.621 1.906-.995 2.431-1.111l-.434-1.953ZM18.338 16.917c1.737.762 3.94.684 5.774.37 1.843-.317 3.56-.918 4.405-1.43l-1.034-1.711c-.555.335-2.004.876-3.71 1.17-1.713.293-3.44.292-4.633-.23l-.802 1.831Zm10.2-1.073c1.666-1.063 2.745-1.8 3.594-2.25.844-.447 1.184-.454 1.407-.387.3.088.722.397 1.516 1.346.776.927 1.72 2.223 3.157 4.064l1.576-1.23c-1.374-1.762-2.398-3.16-3.199-4.117-.782-.935-1.543-1.703-2.48-1.98-1.014-.301-1.965.034-2.914.538-.944.5-2.157 1.325-3.733 2.33l1.076 1.686Zm9.674 2.773c3.939 5.05 8.488 9.969 11.153 12.157l1.27-1.545c-2.5-2.052-6.943-6.838-10.846-11.842l-1.578 1.23ZM18.14 10.2c-1.095.817-1.672 2.17-1.708 3.422-.037 1.259.485 2.673 1.905 3.295l.802-1.831c-.438-.192-.73-.671-.708-1.406.022-.74.374-1.48.905-1.877L18.141 10.2Zm19.73-8.628c-2.498-1-5.39-1.934-8.32-.464l.897 1.787c2.066-1.036 4.18-.466 6.68.534l.743-1.857Zm-8.32-.464c-1.601.803-4.455 2.597-7.029 4.12-1.309.775-2.558 1.489-3.588 2.007-.516.259-.96.46-1.317.594a4.062 4.062 0 0 1-.437.14 1.42 1.42 0 0 1-.132.028c-.033.005-.048.005-.048.005v2c.425 0 .897-.142 1.32-.3.45-.17.965-.406 1.511-.68 1.096-.55 2.394-1.293 3.71-2.073 2.677-1.584 5.387-3.291 6.907-4.054l-.896-1.787ZM17 8.002h-4v2h4v-2Zm-4 23h.83v-2H13v2Zm4.008 1.159c2.196 1.87 4.914 4.23 7.267 6.143 1.177.956 2.28 1.815 3.195 2.453.457.319.882.593 1.256.799.345.19.748.38 1.132.436l.284-1.98c.016.002-.015-.001-.107-.039-.087-.035-.2-.09-.344-.17a12.635 12.635 0 0 1-1.077-.686c-.854-.596-1.913-1.419-3.077-2.365-2.33-1.893-5.013-4.224-7.232-6.114l-1.297 1.523Zm12.85 9.83c1.22.175 2.151.08 2.937-.192.758-.262 1.33-.678 1.761-.966l-1.112-1.663c-.506.339-.841.58-1.303.739-.433.15-1.034.241-2 .103l-.283 1.98Zm4.698-1.158c-.053.035-.056.017.074-.005.12-.02.286-.033.516-.04.414-.013 1.083-.004 1.709-.056 1.303-.108 2.982-.509 4.003-2.217l-1.716-1.027c-.523.875-1.377 1.161-2.453 1.25-.564.048-1.047.033-1.605.05-.255.009-.53.024-.787.068a2.178 2.178 0 0 0-.853.314l1.112 1.663Zm6.302-2.318c.019-.031.035-.086.285-.115.33-.04.667.003 1.274.022.521.017 1.215.009 1.875-.275.716-.306 1.281-.883 1.637-1.775l-1.858-.74c-.18.452-.39.602-.567.677-.231.1-.545.13-1.023.114-.391-.012-1.045-.072-1.573-.01-.608.073-1.316.323-1.766 1.075l1.716 1.027Zm5.07-2.143c.113-.282.22-.333.246-.346.06-.03.16-.05.37-.044l.061-2c-.31-.009-.806.001-1.309.247-.535.261-.956.728-1.225 1.402l1.858.74Zm1.67-.454c1.763-.316 3.46-.997 4.553-1.95.55-.48 1.017-1.092 1.183-1.83.174-.78-.022-1.555-.534-2.238l-1.6 1.2c.207.276.213.462.182.6-.04.18-.186.446-.546.76-.728.635-2.04 1.212-3.591 1.49l.353 1.968Zm4.568-6.404c-.142-.024-.355-.032-.454-.037a5.474 5.474 0 0 1-.447-.04c-.336-.047-.535-.128-.623-.202l-1.284 1.534c.53.442 1.19.587 1.63.648.236.033.452.047.612.056l.234.013.332-1.972ZM39.789 17.387c-1.37-1.757-2.395-3.164-3.186-4.12-.774-.935-1.528-1.71-2.461-1.993-1.013-.308-1.966.03-2.917.534-.946.503-2.169 1.333-3.763 2.35l1.076 1.686c1.684-1.075 2.773-1.817 3.625-2.27.849-.45 1.182-.452 1.398-.387.294.09.71.399 1.5 1.355.775.936 1.711 2.23 3.15 4.075l1.578-1.23Zm-12.306-3.241c-.548.33-1.999.871-3.708 1.164-1.72.295-3.448.296-4.635-.224l-.802 1.831c1.732.76 3.94.68 5.775.365 1.845-.317 3.564-.916 4.404-1.425l-1.034-1.711Zm-8.343.94c-.438-.192-.729-.671-.707-1.406.022-.742.375-1.482.904-1.877L18.141 10.2c-1.093.816-1.67 2.169-1.707 3.42-.038 1.259.483 2.675 1.904 3.297l.802-1.831Zm.189-3.277c1.837-1.342 4.558-3.416 7.191-5.25 1.313-.916 2.583-1.757 3.686-2.403 1.134-.664 1.992-1.063 2.511-1.178l-.434-1.953c-.855.19-1.95.739-3.088 1.405-1.168.685-2.487 1.56-3.818 2.487-2.655 1.85-5.44 3.971-7.228 5.277l1.18 1.615Zm30.043-3.582c-2.014-1.644-4.97-3.699-8.036-5.217-1.533-.76-3.118-1.396-4.643-1.775-1.517-.377-3.037-.515-4.41-.21l.434 1.953c.96-.214 2.143-.138 3.493.198 1.343.333 2.788.908 4.239 1.627 2.903 1.437 5.734 3.402 7.657 4.973l1.266-1.549Zm2.792 23.26 3-.499-.328-1.973-3 .498.328 1.973ZM56 30.002v-20h-2v20h2Zm-6.642.767c.529.442 1.191.586 1.63.648.236.033.452.047.612.056l.234.013.332-1.972c-.141-.024-.354-.032-.452-.037a5.425 5.425 0 0 1-.449-.04c-.337-.048-.536-.13-.623-.202l-1.284 1.534ZM50.74 9.002c-.274 0-.42-.056-.564-.142-.214-.128-.383-.29-.803-.633l-1.266 1.549c.614.501 1.29 1.226 2.633 1.226v-2Zm4.261 0h-4.261v2H55v-2Zm-51-3h4v-2H4v2Zm4 0h2v-2H8v2Zm27.246-3.106 1.98.566.549-1.923-1.98-.566-.549 1.923Zm-2.429.054a4.104 4.104 0 0 1 2.429-.054l.55-1.923a6.104 6.104 0 0 0-3.613.08l.634 1.897Zm13.726 33.032c.318.01.68.003 1.055-.064l-.353-1.968c-.184.032-.39.04-.64.033l-.062 1.999Zm-32.713-4.98c1.17 0 2.296.407 3.178 1.159l1.297-1.523a6.892 6.892 0 0 0-4.475-1.636v2Z"
            fill={
              {
                "payload": 4282006074,
                "type": 0,
              }
            }
            propList={
              [
                "fill",
              ]
            }
          />
          <RNSVGCircle
            cx="7"
            cy="11.002"
            fill={
              {
                "payload": 4290513165,
                "type": 0,
              }
            }
            propList={
              [
                "fill",
              ]
            }
            r="2"
          />
          <RNSVGCircle
            cx="61"
            cy="27.002"
            fill={
              {
                "payload": 4290513165,
                "type": 0,
              }
            }
            propList={
              [
                "fill",
              ]
            }
            r="2"
          />
        </RNSVGGroup>
      </RNSVGSvgView>
      <Text
        style={
          [
            {
              "fontFamily": "ReservaSerif-Bold",
              "fontSize": 15,
            },
            {
              "fontSize": 20,
              "lineHeight": 20,
              "marginBottom": 16,
              "textAlign": "center",
            },
          ]
        }
        variant="descontoTag2"
      >
        Programa de Parcerias
      </Text>
      <Text
        style={
          [
            {
              "fontFamily": "Nunito-Regular",
              "fontSize": 15,
            },
            {
              "color": "#848484",
              "fontSize": 16,
              "lineHeight": 20,
              "textAlign": "center",
            },
          ]
        }
        variant="tituloSessao"
      >
        <Text
          style={
            [
              {
                "fontFamily": "Nunito-Bold",
                "fontSize": 20,
              },
              {
                "color": "#000000",
                "fontSize": 16,
                "lineHeight": 20,
              },
            ]
          }
          variant="precoTotal"
        >
          Acesso exclusivo
        </Text>
         
        a benefícios com marcas parceiras de variados segmentos.
      </Text>
    </View>
  </View>
</View>
`;
